lib/core/src/main/java/io/atlasmap/core/ADMArchiveHandler.java

```diff
@@ -720,6 +720,21 @@ public void deleteDocument(DataSourceType dsType, String documentId) throws Atla
         getAtlasMappingHandler().removeDocumentReference(docKey);
     }
 
+    /**
+     * Delete the specified field from the specified mapping.
+     *
+     * @param dsType
+     * @param mappingId
+     * @param fieldIndex
+     * @throws AtlasException
+     */
+    public void deleteMappingField(DataSourceType dsType, String mappingId, Integer fieldIndex) throws AtlasException {
```

Comment:
We should directly invoke `AtlasMappingHandler#removeMappingField()` from `MappingService`, instead of adding this to `ADMArchiveHandler`.

Reply:
done